config/secrets.json.example

```diff
@@ -1,30 +1,25 @@
 {
   // Twitter API configuration
-  "twitter": {
-    "consumer_key": "xxx",
-    "consumer_secret": "xxx",
-    "access_token": "xxx",
-    "access_token_secret": "xxx"
-  },
+  // "twitter": {
+  //   "consumer_key": "xxx",
+  //   "consumer_secret": "xxx",
+  //   "access_token": "xxx",
+  //   "access_token_secret": "xxx"
+  // },
 
   // Facebook API configuration
-  // To obtain token, do GET https://graph.facebook.com/oauth/access_token?client_secret=xxx&client_id=xxx&grant_type=client_credentials
-  // Obtain client_id and client_secret from https://developers.facebook.com/apps/ (create an app if necessary)
-  "facebook": {
-    "accessToken": "xxx"
-  },
+  // "facebook": {
+  //   "accessToken": "xxx"
+  // },
 
   // ELMO API configuration
-  "elmo": {
-    "authToken": "xxx"
-  },
+  // "elmo": {
+  //   "authToken": "xxx"
+  // },
 
   // MongoDB database connection parameters
   "mongodb": {
-    "username": "username",
-    "password": "password",
```

Comment:
Automatically generated. Removed to reduce confusion for someone just setting up the project.